platform-includes/logs/setup/android.mdx

```diff
@@ -32,3 +32,7 @@ class MyApplication : Application() {
   }
 }
 ```
+
+```XML {filename:AndroidManifest.xml} {12}
```

Comment:
I think the code highlighting can be removed since its only one line or is that there on purpose

Reply:
TIL, I thought it was some kind of ordering of the tabs or something 😅